sycl/doc/extensions/experimental/sycl_ext_intel_data_flow_pipes_properties.asciidoc

```diff
@@ -103,16 +103,19 @@ value to determine which of the extension's APIs the implementation supports.
 Below is a list of compile-time-constant properties which `pipe` supports.
 
 ```c++
-namespace sycl::ext::intel::experimental {
+namespace sycl {
+namespace ext {
+namespace intel {
+namespace experimental {
 
 struct ready_latency_key {
-  template <int Latency>
+  template <unsigned Latency>
```

Comment:
To avoid some potential headaches, maybe we should use fixed-size integers here, i.e. `uint32_t`.